mock/server.py

```diff
@@ -165,13 +169,14 @@ def genVolserId():
 }
 @app.route('/login/', methods=['POST'])
 @app.route('/login', methods=['POST'])
+
 def login():
     if request.get_json()['username'] == global_username:
         if request.get_json()['password'] == global_password:
             if global_password_expired == False:
                 resp = make_response("Login Successful")
-                resp.set_cookie("jedHTTPSession", "xrQdqvc2J7WWlew2OXU1RtwwUXXD9KGJ35x5IZTrSrX18y80OVBI8A")
-                return resp
+                resp.set_cookie("jedHTTPSession." + global_port, "xrQdqvc2J7WWlew2OXU1RtwwUXXD9KGJ35x5IZTrSrX18y80OVBI8A")
```

Comment:
Putting a port here works in some cases, but not the default case.
A better default is to use the env var "ZWE_zowe_cookieIdentifier" which defaults to "1" in a real zowe install.
https://github.com/zowe/zowe-install-packaging/blob/v2.x/master/example-zowe.yaml#L301=

Reply:
So in real-time, the cookie name would be starting with jedHTTPSession.1? But this could cause an error with the mock server since the mock server needs a specific port number after the dot "."  .

Reply:
got it. Should I change it to .1 instead in server.py?

Reply:
> yes, it would be jedHTTPSession.1
Here's how it works in ZSS: https://github.com/zowe/zss/blob/v2.x/staging/c/zss.c#L1171-L1175
The app-server uses this same logic, so basically the app-server may be expecting "jedHTTPSession.1" rather than "jedHTTPSession.5000" depending upon environment vars & config files.
You can still set the mock server to use port 5000, I'm just saying that in a real zowe install, zss wouldn't be presenting "jedHTTPSession.7557" (7557 is the default port), it would be presenting "jedHTTPSession.1"

The ".1" case I believe is the HA case. For non-HA mode, cookie is the ZSS port. 
from zssHandler.js in sso-auth
`const zssPort = serverConf.agent.https && serverConf.agent.https.port ? serverConf.agent.https.port : serverConf.agent.http.port;
    this.zssCookieName = zluxUtil.isHaMode() ? COOKIE_NAME_BASE + zoweInstanceId : COOKIE_NAME_BASE + zssPort;`

Reply:
That's why when we use mock server, if used in the regular development app server environment case, would be in this way - otherwise default mock server case this doesn't successfully log in

Reply:
```
const isHaMode = module.exports.isHaMode = function () {
  const haInstanceCount = +process.env['ZWE_HA_INSTANCES_COUNT'];
  return (!isNaN(haInstanceCount) && haInstanceCount > 1);
}
```

we can check for HA case, ok